src/riak_kv_qry_compiler.erl

```diff
@@ -97,16 +97,22 @@ run_select(Select, Row, InitialState) ->
 
 %% @priv
 run_select2([], _, _, Acc) ->
```

Comment:
@lehoff how much slower than your version?  If we have numbers that show a different implementation improves throughput we need to put it in ASAP.